src/vs/base/test/browser/ui/grid/gridview.test.ts

```diff
@@ -22,7 +22,7 @@ suite('Gridview', function () {
 	});
 
 	teardown(function () {
-		gridview = null;
+		gridview = null!;
```

Comment:
Actually the right thing to do here is to just remove the whole `teardown` section and simply never set `gridview` to `null` and let the garbage collector do its thing.